apps/condo/domains/billing/schema/resolvers/resolver.js

```diff
@@ -0,0 +1,120 @@
+const { isEmpty, isNil, xorWith, isEqual } = require('lodash')
+
+const { GQLError } = require('@open-condo/keystone/errors')
+const { getLogger } = require('@open-condo/keystone/logging')
+
+const isArrayEqual = (x, y) => isEmpty(xorWith(x, y, isEqual))
+
+const appLogger = getLogger('condo')
+
+class Resolver {
```

Comment:
it looks too complex. JavaScript is not Java. Try not to use inheritance. It's better to create util functions and reuse them rather than create complex layered inheritance logic.